src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
```

Comment:
checklist item 7
https://github.com/mate-academy/layout_catalog/blob/master/checklist.md

Reply:
Main item was not fixed. BEM-blocks should not have margins. use mix.